src/patternfly/components/Table/table-grid.scss

```diff
@@ -126,7 +113,7 @@
     display: block;
 
     &:first-of-type {
-      border-top: var(--pf-c-table-tbody--responsive--BorderWidth) solid var(--pf-c-table--responsive--BorderColor);
+      border-top: var(--pf-c-table--m-grid--tbody--BorderWidth) solid var(--pf-c-table--m-grid--BorderColor);
```

Comment:
border-top seems to be missing:

<img width="645" alt="Screen Shot 2020-03-20 at 3 03 49 PM" src="https://user-images.githubusercontent.com/20118816/77197732-14961f00-6abc-11ea-83be-2b7099151ef4.png">